docs/source/app-dev/services.rst

```diff
@@ -94,6 +94,17 @@ The command submission service deduplicates submitted commands based on their :r
 
 For details on how to use command deduplication, see the :ref:`Application Architecture Guide <command-deduplication>`.
 
+.. note:: Ledgers with command deduplication across participants
+
+  - The ledger server may extend the deduplication period specified in the request arbitrarily, up until the maximum deduplication duration specified in the :ref:`ledger configuration <ledger-configuration-service>`.
+    The deduplication period chosen by the ledger server is the *effective deduplication period*.
+
+  - Regardless, the deduplication period specified in the request is always checked against the configured maximum deduplication duration.
+
+  - A command submission is considered a duplicate submission if the ledger server is aware of another command within the *effective* deduplication period and with the same :ref:`change ID <change-id>`.
+
+  - Command deduplication is *guaranteed* to work even if commands are submitted to different participants.
```

Comment:
Is this true for VMBC and Canton? Maybe we can just mention both in this sentence.

Reply:
Sounds good. This is certainly true for VMBC, @andreaslochbihler-da can comment on Canton.

Reply:
@gerolf-da Opted for a slightly different presentation (and left out Canton till I get a confirmation) -- let me know what you think.

Reply:
Command deduplication is formally defined via the completion event stream. Since Canton produces a completion event only on the participant that made the submission, Canton doesn't deduplicate commands across participants.

Reply:
OK, removed it.